test/e2e/runner/evidence.go

```diff
@@ -179,6 +190,10 @@ func generateLightClientAttackEvidence(
 		return nil, err
 	}
 
+	if !validEvidence {
```

Comment:
nit: could potentially add a comment in the style of "malleate the last signature of the commit by adding one to its first byte"

Reply:
👍